packages/bpk-component-boilerplate/readme.md

```diff
@@ -0,0 +1,28 @@
+# bpk-component-boilerplate
+
+> Backpack example component.
+
+## Installation
+
+```sh
+npm install bpk-component-boilerplate --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkBoilerplate from 'bpk-component-code';
+
+export default () => (
+  <div>
```

Comment:
Why does this needs a wrapping `<div />`

Reply:
Cos I'm so extra

(I'll remove it)